src/features/donations/screens/TreeDonation.tsx

```diff
@@ -93,6 +93,7 @@ function TreeDonation({
     }
 
     setMinAmt(getMinimumAmountForCurrency(currency));
+    console.log(minAmt);
```

Comment:
Please remove unnecessary console.log statements again once this PR is ready to get merged.

Reply:
removed